src/pages/registerForm.js

```diff
@@ -0,0 +1,110 @@
+import { Form, Input, Button} from 'antd';
+import { Link, useNavigate } from 'react-router-dom';
+import { routes } from '../config/routes';
+import { httpsService } from '../utils/https.service';
+import '../styles/registerPage.scss';
+
+const formItemLayout = {
+  labelCol: {
+    xs: {
+      span: 24,
+    },
+    sm: {
+      span: 8,
+    },
+  },
+  wrapperCol: {
+    xs: {
+      span: 24,
+    },
+    sm: {
+      span: 16,
+    },
+  },
+};
+const tailFormItemLayout = {
+  wrapperCol: {
+    xs: {
+      span: 24,
+      offset: 0,
+    },
+    sm: {
+      span: 16,
+      offset: 8,
+    },
+  },
+};
+
+export const RegisterForm = () => {
+  const [form] = Form.useForm();
+  const navigate = useNavigate();
+  
+
+  const handleRegister = async (body) => {
+      
+    //console.log(body);
+
+    try {
+        const response = await httpsService.post('/user', body);
+        console.log(response);
+    } catch (e) {
+        console.log('fatalerror')
+    }
+    
+
+    navigate(routes.homepage);
```

Comment:
Do we need to navigate in spite of success of request? This will work for both success and error, but in case of error we need to show notification only in my opinion

Reply:
BTW where are notifications in handlers?

Reply:
agree
remove navigation, added error notification